### PR TITLE
chore: rename settings keys, remove keys

### PR DESCRIPTION
**Pull request by ftreguer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ Unikname is member of the W3C and the DIF (Decentralized Identity Foundation) an
 
 ## Support / Reporting an Issue
 
-Before to [report en issue in github](https://github.com/unik-name/discourse-unikname/issues) you're welcome to [open a discussion with the support on our Forum](https://forum.unikname.com/c/support/15) (a Discourse one ;-) 
+Before to [report en issue in github](https://github.com/unik-name/discourse-unikname-connect-authentication-plugin/issues) you're welcome to [open a discussion with the support on our Forum](https://forum.unikname.com/c/support/15) (a Discourse one ;-)
```
Comment (dlecan) [inline README.md]: Relative link?

File: config/settings.yml [hunk 2]
```diff
@@ -2,27 +2,13 @@ plugins:
   unikname_connect_enabled:
     default: false
     client: true
-  unikname_connect_discovery_document:
-    default: "https://connect.unikname.com/oidc/.well-known/openid-configuration"
-  unikname_connect_unikname_key:
+  unikname_connect_Business_Account_ID:
```
Comment (dlecan) [inline config/settings.yml]: lowercased

File: config/settings.yml [hunk 3]
```diff
@@ -2,27 +2,13 @@ plugins:
...
     default: ""
-  unikname_connect_secret:
+  unikname_connect_API_secret_key:
```
Comment (dlecan) [inline config/settings.yml]: lowercased

File: lib/unikname_connect_authenticator.rb [hunk 4]
```diff
@@ -51,17 +51,17 @@ def register_middleware(omniauth)
         opts = env['omniauth.strategy'].options
 
         token_params = {}
-        token_params[:scope] = SiteSetting.unikname_connect_token_scope if SiteSetting.unikname_connect_token_scope.present?
+        token_params[:scope] = ""
```
Comment (dlecan) [inline lib/unikname_connect_authenticator.rb]: missing `openid` here?

**Review by dlecan — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ftreguer) [reply README.md]: ?

**Review by dlecan — APPROVED** [review]

**Pull request merged**